src/ReferenceOverlapMap.hpp

```diff
@@ -1,6 +1,12 @@
 #ifndef SHASTA_REFERENCEOVERLAPMAP_HPP
 #define SHASTA_REFERENCEOVERLAPMAP_HPP
 
+/// The overlap map is based on a boost interval map. The interval map performs an "aggregation" operation whenever
```

Comment:
I meant we need a higher level comment describing how we use these mappings to the reference when displaying the alignment candidate graph. Does not have to be super detailed - a general idea is sufficient.

Reply:
ok, check out the latest commit